nixos/modules/services/torrent/rtorrent.nix

```
@@ -203,6 +203,30 @@ in {
             ExecStart="${cfg.package}/bin/rtorrent -n -o system.daemon.set=true -o import=${rtorrentConfigFile}";
             RuntimeDirectory = "rtorrent";
             RuntimeDirectoryMode = 755;
```

```suggestion
            RuntimeDirectoryMode = 750;
```

Maybe not world-readable? And add a UMask.

---

Done.

Umask is already set in `rtorrent` itself: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/torrent/rtorrent.nix#L164.